src/components/cron.ts

```diff
@@ -126,3 +132,28 @@ export const createCoffeeChatCron = (client: Client): CronJob =>
       throw 'Bad channel type';
     }
   });
+
+export const assignCodeyRoleForLeaderboard = (client: Client): CronJob =>
+  new CronJob('0 0 0 */1 * *', async function () {
+    const leaderboard = await getCoinLeaderboard(numberUsersToAssignRole);
+    const guild = client.guilds.resolve(TARGET_GUILD_ID);
+    if (!guild) {
+      throw new CodeyUserError(undefined, 'guild not found');
+    }
+    console.log('running cron job');
+    const members = await guild.members.fetch();
+    // Removing role from previous users
+    const usersPreviousRole = await loadRoleUsers(CODEY_COIN_ROLE_ID);
+    const guildMembersPreviousRole = usersPreviousRole.map((user) => members.get(user.id));
```

Comment:
Make a new helper function called `loadRoleMembers`. `loadRoleUsers` gets the members, maps them to users, then It is redundant to map them back to members here. You can change `loadRoleUsers` to use `loadRoleMembers`. Also, ensure that `loadRoleMembers` and `loadRoleUsers` both use camel case for the variable names.